src/writer/creator.cpp

```diff
@@ -88,12 +88,11 @@ namespace zim
 
       for(unsigned i=0; i<nbWorkerThreads; i++)
       {
-        pthread_t thread;
-        pthread_create(&thread, NULL, taskRunner, this->data.get());
-        data->workerThreads.push_back(thread);
+        std::thread thread(taskRunner, this->data.get());
+        data->workerThreads.push_back(std::move(thread));
       }
 
-      pthread_create(&data->writerThread, NULL, clusterWriter, this->data.get());
+      data->writerThread = std::move(std::thread(clusterWriter, this->data.get()));
```

Comment:
I believe that `std::move()` is not required here.

Reply:
Yes, I've removed it.